aviatrix/resource_aviatrix_firewall.go

```diff
@@ -363,7 +383,7 @@ func resourceAviatrixFirewallUpdate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	d.Partial(false)
-	return nil
+	return resourceAviatrixFirewallRead(d, meta)
```

Comment:
use the deferred read since you have other exit points above

Reply:
Deferred function will be evaluated, maybe will need to be added to other resources as well.